third_party/mavlink/mavlink.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 92ed5676..78971ca9 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -2,6 +2,16 @@ cmake_minimum_required(VERSION 3.13)
+ 
+ project(mavlink)
+ 
++# Needed to find the right Python on macOS
++if (NOT DEFINED Python_FIND_FRAMEWORK)
++    set(Python_FIND_FRAMEWORK "LAST")
++endif()
++
++# Needed to find the right Python on Windows
++if (NOT DEFINED Python_FIND_REGISTRY)
++    set(Python_FIND_REGISTRY "LAST")
++endif()
++
```

Comment:
So that's a patch for now, I don't know if it should go upstream or not. I need that for the github actions to work (otherwise Windows/macOS don't detect the right Python version, and so `pip install future` has not been run for the one it picks).

The thing is that I tried manually on my Windows and I don't need that: I can just `pip install future` and then it works...